frontend/src/components/Modal/ProfileModal/ProfileModal.tsx

```diff
@@ -10,13 +10,22 @@ import { getProfilesForLoggedUser } from "store/profile";
 import { inputsArray } from "./InputsArray";
 import { ProfileModalProps } from "interfaces";
 import { AxiosResponse } from "axios";
+import { ProfilePicture } from "components/profilePicture/ProfilePicture";
```

Comment:
I believe that path "components" will be enough.

Reply:
Yes, provided you have index.ts that exports this function then path can be shortened